Tests/OpenCombineTests/PublisherTests/DeferredTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  DeferredTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 7/7/19.
+//
+
+import XCTest
+
+//import Combine
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class DeferredTests: XCTestCase {
+
+    static let allTests = [
+        ("testDeferredNotCalledBeforeSubscription",
+            testDeferredNotCalledBeforeSubscription),
+        ("testDeferredCreatedAfterSubscription",
+            testDeferredCreatedAfterSubscription)
+    ]
+
+    func testDeferredNotCalledBeforeSubscription() {
+        let subscription = CustomSubscription()
+        let publisher = CustomPublisher(subscription: subscription)
+
+        _ = Publishers.Deferred {
+            publisher
+        }
+
+        XCTAssertEqual(subscription.history, [])
+    }
+
+    func testDeferredCreatedAfterSubscription() {
+        var deferredCalled = false
```

Comment:
Le's make sure that the closure is called as many times as `Publishers.Deferred` receives a subscriber, and the publisher is not cached instead. Probably better use a counter instead of a `Bool` variable